OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -78,28 +76,12 @@ class OSIMSIMULATION_API Joint : public ModelComponent {
 OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
 
 public:
-
 //==============================================================================
 // PROPERTIES
 //==============================================================================
-    OpenSim_DECLARE_PROPERTY(location_in_parent, SimTK::Vec3,
-        "Location of the joint in the parent body specified in the parent "
-        "reference frame. Default is (0,0,0).");
-
-    OpenSim_DECLARE_PROPERTY(orientation_in_parent, SimTK::Vec3,
-        "Orientation of the joint in the parent body specified in the parent "
-        "reference frame. Euler XYZ body-fixed rotation angles (in radians) "
-        "are used to express the orientation. Default is (0,0,0).");
-
-    OpenSim_DECLARE_PROPERTY(location_in_child, SimTK::Vec3,
-        "Location of the joint in the child body specified in the child "
-        "reference frame. Default is (0,0,0).");
-
-    OpenSim_DECLARE_PROPERTY(orientation_in_child, SimTK::Vec3,
-        "Orientation of the joint in the child body specified in the child body "
-        "reference frame. Euler XYZ body-fixed rotation angles (in radians) "
-        "are used to express the orientation. Default is (0,0,0)" );
-
+    /** @name Property declarations
+    These are the serializable properties associated with a Joint. **/
+    /**@{**/
```

Comment:
Remove this block (and the corresponding end `/**@}**/`); they are no longer necessary.